satpy/readers/clavrx.py

```diff
@@ -70,6 +70,7 @@
 
 def _get_sensor(sensor: str) -> str:
     """Get the sensor."""
+    LOG.debug('SENSORS: {}'.format(sensor))
```

Comment:
an f-string would be more concise, eg
```suggestion
    LOG.debug(f'SENSORS: {sensor}')
```